apache-hbase-mixin/dashboards/apache-hbase-cluster-overview.libsonnet

```diff
@@ -0,0 +1,1032 @@
+        ],
+      },
+    },
+    overrides: [],
+  },
+  options: {
+    colorMode: 'value',
+    graphMode: 'none',
+    justifyMode: 'auto',
+    orientation: 'auto',
+    reduceOptions: {
+      calcs: [
+        'lastNotNull',
+      ],
+      fields: '',
+      values: false,
+    },
+    textMode: 'value',
+  },
+  pluginVersion: '10.2.0-61719',
+};
+
+local regionserversPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'server_num_region_servers{job=~"$job", hbase_cluster=~"$hbase_cluster"}',
+      datasource=promDatasource,
+      legendFormat='{{hbase_cluster}}',
+      format='table',
+    ),
+    prometheus.target(
+      'server_num_reference_files{job=~"$job", hbase_cluster=~"$hbase_cluster"}',
+      datasource=promDatasource,
+      format='table',
+    ),
+  ],
+  type: 'table',
+  title: 'RegionServers',
+  description: 'RegionServers for a cluster.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'palette-classic',
+      },
+      custom: {
+        align: 'left',
+        cellOptions: {
+          type: 'auto',
+        },
+        inspect: false,
+      },
+      links: [],
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+        ],
+      },
+    },
+    overrides: [
+      {
+        matcher: {
+          id: 'byName',
+          options: 'Instance',
+        },
+        properties: [
+          {
+            id: 'links',
+            value: [
+              {
+                title: '',
+                url: '/d/apache-hbase-regionserver-overview',
+              },
+            ],
+          },
+          {
+            id: 'mappings',
+            value: [],
+          },
+        ],
+      },
+      {
+        matcher: {
+          id: 'byName',
+          options: 'Is master',
+        },
+        properties: [
+          {
+            id: 'noValue',
+            value: 'false',
+          },
+        ],
+      },
+    ],
+  },
+  options: {
+    cellHeight: 'md',
+    footer: {
+      countRows: false,
+      fields: '',
+      reducer: [
+        'sum',
+      ],
+      show: false,
+    },
+    showHeader: true,
+    sortBy: [],
+  },
+  pluginVersion: '10.2.0-61719',
+  transformations: [
+    {
+      id: 'merge',
+      options: {},
+    },
+    {
+      id: 'organize',
+      options: {
+        excludeByName: {
+          Time: true,
+          'Time 1': true,
+          'Time 2': true,
+          Value: true,
+          'Value #A': true,
+          'Value #B': true,
+          __name__: true,
+          '__name__ 1': true,
+          '__name__ 2': true,
+          clusterid: true,
+          'clusterid 1': true,
+          'clusterid 2': true,
+          context: true,
+          'context 1': true,
+          'context 2': true,
+          hbase_cluster: true,
+          'hbase_cluster 1': true,
+          'hbase_cluster 2': true,
```

Comment:
Should the RegionServers chart be larger and display which HBase cluster it belongs to? I am guessing each cluster has 1 master, which the Hbase RegionServer overview looks at those master stats then?

Reply:
I'd be in favor of expanding the table, just need to figure out what else to re-size. Master stats are actually shown on the cluster overview, not on the RegionServer dashboard.